### PR TITLE
Top malicious ratio indicators

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by meirwah** [comment]
@erezh31 build fails:
```
Failed: Scripts/script-TopMaliciousRatioIndicators.yml failed
<SchemaError: error code 2: Schema validation failed:
 - Key 'fromVersion' was not defined. Path: ''.: Path: '/'>
```

**Comment by erezh31** [comment]
@asafshen I updated "script" schema validation. can you verify it? I think it was a bug

**Comment by asafshen** [comment]
@erezh31 the schema validation is case sensitive (we cannot configure this to be insensitive)

I recommend to:
 - revete script schema yml
 - in your script - change the from/to version to match the schema (`fromversion` `toversion`)

BTW -- ideally, it should be identical in all artifacts, unfortunately it is not

**Comment by erezh31** [comment]
@meirwah CI failed on some playbook, not related to the script
can you plz review and merge?

File: Scripts/script-TopMaliciousRatioIndicators.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+commonfields:
+  id: TopMaliciousRatioIndicators
+  version: -1
+fromversion: "4.0.0"
+name: TopMaliciousRatioIndicators
+script: |
+  from datetime import datetime, timedelta
+  import json, random
+
+  MAX_INDICATORS = 10000
```
Comment (meirwah) [inline Scripts/script-TopMaliciousRatioIndicators.yml]: make this an argument

File: Scripts/script-TopMaliciousRatioIndicators.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  MIN_NUMBER_OF_INVS = 3
```
Comment (meirwah) [inline Scripts/script-TopMaliciousRatioIndicators.yml]: make this an argument

File: Scripts/script-TopMaliciousRatioIndicators.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  demisto.results(json.dumps({"total": len(sorted_indicators), "data": sorted_indicators}))
+type: python
+tags: []
+comment: |-
+  Find the top malicious ratio indicators.
+  Malicious ratio is defined by the ratio between the number of "bad" incidents divided by the number of total number of incidents that the indicators appears in.
+enabled: true
+scripttarget: 0
+runonce: false
+releaseNotes: "Add new script"
```
Comment (meirwah) [inline Scripts/script-TopMaliciousRatioIndicators.yml]: make this adding malicious ratio indicators script

**Comment by meirwah** [comment]
@erezh31 plz fix and merge

File: Scripts/script-TopMaliciousRatioIndicators.yml (hunk 1 above)
Comment (erezh31) [reply Scripts/script-TopMaliciousRatioIndicators.yml]: It's for dashboard use - we can't really set custom argument. we still want to add this?

File: Scripts/script-TopMaliciousRatioIndicators.yml (hunk 2 above)
Comment (erezh31) [reply Scripts/script-TopMaliciousRatioIndicators.yml]: It's for dashboard use - we can't really set custom argument. we still want to add this?

File: Scripts/script-TopMaliciousRatioIndicators.yml (hunk 1 above)
Comment (meirwah) [reply Scripts/script-TopMaliciousRatioIndicators.yml]: if this is a dashboard script, aren't you missing:
```
tags:
- widget
```
?

**Comment by erezh31** [comment]
@meirwah added the widget and tested together.
plz review the widget and changes

**Comment by meirwah** [comment]
fix build first @erezh31 : 
image

**Comment by erezh31** [comment]
@meirwah fixed

**Pull request merged**